vms/platformvm/service.go

```diff
@@ -674,6 +676,10 @@ func (service *Service) Sign(_ *http.Request, args *SignArgs, reply *SignRespons
 	}
 	user := user{db: db}
 
+	if args.Signer.IsZero() {
```

Comment:
Can you move the argument validity checks to the top of the method?
No use in doing work and then throwing it out because an arg is invalid.

Reply:
For sure, I'll push the change.